packages/core/src/PlayerContextProvider.js

```diff
@@ -700,7 +711,7 @@ export class PlayerContextProvider extends Component {
         index = playlist.length - 1;
       }
     }
-    this.goToTrack(index);
+    this.goToTrack(index, true, true);
```

Comment:
Using the hash for arguments will allow us to avoid explicitly passing `shouldPlay: true`.